v2x/vlog_to_pbtype.py

```diff
@@ -345,12 +377,12 @@ def pin_sort(p):
 
 
 def mode_interconnects(mod, mode_name) -> List[(CellPin)]:
```

Comment:
Need to update the type in `mode_interconnects` return type `List[(CellPin)]`.

Reply:
Fixed.